src/main/java/org/generationcp/middleware/manager/WorkbenchDataManagerImpl.java

```diff
@@ -533,4 +533,7 @@ public Role getRoleById(final Integer id) {
 		}
 	}
 
+	public void saveOrUpdateProjectUserInfo(final ProjectUserInfo projectUserInfo) {
```

Comment:
It should be located in a service without exposing the entity class. Use a DTO or send 2 params to the function

Reply:
Is this function being used? I could not find it in BMSAPI